tests/uptane_vector_tests.cc

```diff
@@ -45,67 +45,100 @@ class Uptane_Vector_Test {
   }
 
  public:
-  static bool run_test(const std::string& test_name, const Json::Value& vector, const std::string& port) {
-    std::cout << "VECTOR: " << vector;
+  static bool run_test(const std::string& test_name, const std::string& address) {
     TemporaryDirectory temp_dir;
     Config config;
     config.provision.primary_ecu_serial = "test_primary_ecu_serial";
     config.provision.primary_ecu_hardware_id = "test_primary_hardware_id";
-    config.uptane.director_server = "http://127.0.0.1:" + port + "/" + test_name + "/director";
-    config.uptane.repo_server = "http://127.0.0.1:" + port + "/" + test_name + "/image_repo";
+    config.uptane.director_server = address + test_name + "/director";
+    config.uptane.repo_server = address + test_name + "/image_repo";
     config.storage.path = temp_dir.Path();
-    config.storage.uptane_metadata_path = BasedPath(port + "/aktualizr_repos");
+    config.storage.uptane_metadata_path = BasedPath(address + "aktualizr_repos");
```

Comment:
+1 for reusing `address` as much as possible and avoiding the duplicated parts, but does this actually work for the `uptane_metadata_path`? Would it be better if were actually be a subdir of `temp_dir`?